vortexor/Readme.md

```diff
@@ -145,3 +145,80 @@ Operators can decide whether to adopt Vortexors without concerns about network
 protocol changes. Upgrading involves specifying the Vortexor's TPU address and
 verified packet receiver network address via CLI or Admin RPC. The transition is
 designed to be seamless for operators.
+
+# CLI Setup to pair a vortexor and a validator
+
+On the validator side, configure the vortexor receiver address and the TPU and
+TPU forward address to those of the vortexor: using the following parameters:
+
+* --tpu-vortexor-receiver-address: set the address where the validator is receiving
+the verified packets from the vortexor.
+
+For example when the validator is running on 10.138.0.136 and use port 8100 to
+receive the verified packets:
+
+--tpu-vortexor-receiver-address 10.138.0.136:8100
```

Comment:
why not just port if this is where the validator is receiving it?

Reply:
it has more control on which interface to receive the packets in the case there are multiple interfaces.